lib/_sidebar-template.erb

```diff
@@ -23,6 +27,15 @@
                         <%= "<%- end -\%\>" -%>
 
                             <li class='doc-link__category <%= "<%= active_keywords[0] -\%\>" -%>'>
+                                <svg class="doc-link__exapnd">
```

Comment:
Just a very minor typo on the class name here, it should be `expand` instead of `exapnd`